### ISSUE TITLE
Make the app more accessible - content description

### ISSUE THREAD

**Issue by neslihanturan** [commons-app/apps-android-commons#4471]
**Summary:** 
There are plenty of ways to increase accessibility of our app. One of them is adding content descriptions to UI elements, see [docs](https://developer.android.com/guide/topics/ui/accessibility/apps#describe-ui-element). Currently most of our UI elements does not have their content description. This is a beginner friendly issue. Here would be steps:
- Adding UI element descriptors by following the guideline that I linked.
- Adding them into strings xml so that they will be translated
- Placing them at very top of strings xml so that they won't be confused with other strings, as:
```
<!--accessibility UI description strings-->
  <string name="xxx1">yyy1</string>
  <string name="xxx2">yyy2</string>
  .....
<!--Other strings-->
  <string name="aaa1">bbb1</string>
  <string name="aaa2">bbb2</string>
 ```

**Comment by aileenpalafox** [comment]
Hello, I would like to contribute to this if that's OK. It's my first time on open source so, if it's possible, I would like some more guidance on how to do this. :)

**Comment by TheProgrammerGod** [comment]
Hey @neslihanturan , could you assign this issue to me? I'd to contribute to this as my first open-source contribution.

**Comment by mariammali273** [comment]
I'd like to contribute to this, it's my first time contributing to open-source projects.

**Comment by reshmanjali** [comment]
Hey!
I'd like to contribute to this, it's my first time contributing to open-source projects. Appreciate any guidance.

**Comment by neslihanturan** [comment]
Hi, pinging @mariammali273 and @TheProgrammerGod to see if they are still interested. If they are not, it is yours @reshmanjali . We can wait 2 days to hear from them. Please make sure you have read our contribution guide docs meanwhile.

**Comment by Anand-Avinash-Bhalerao** [comment]
Hey, I would like to contribute to this. This is my first time contributing to open source.

**Comment by Rishavgupta12345** [comment]
I would like to work on this issue PLease assign me

**Comment by nicolas-raoul** [comment]
@Rishavgupta12345 Are you still willing to work on this? If yes, you might want to test Ben's pull request? :-)

By the way, I just tested Accessibility Scanner on our app, the results are interesting. Example:
![Screenshot_20220524-132815](https://user-images.githubusercontent.com/99590/169949571-b30d5761-37c1-42ff-8b14-1e61b75577c5.png)
